O/OpenPolicyAgent/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "OpenPolicyAgent"
+version = v"0.55.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/open-policy-agent/opa.git", "4c21dddf2254632e8b28b60b73863f22bd3ae389")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd opa/
+WASM_ENABLED=0 make build
+mkdir -p $bindir
+ls -la opa_*
+cp opa_* $bindir/opa${exeext}
```

Comment:
To make it slightly clearer
```suggestion
install -Dvm 755 opa_* $bindir/opa${exeext}
```